terraform/environments/bootstrap/member-bootstrap/providers.tf

```diff
@@ -11,3 +11,11 @@ provider "aws" {
   alias  = "modernisation-platform"
   region = "eu-west-2"
 }
+# AWS provider (workspace): the workspace account. Required for assuming a role into an account for bootstrapping
```

Comment:
This provider isn't needed, the boot strapping won't be creating any resources in us-east-1, also I think this would create a dependancy loop if you used this provider as you are creating a role then trying to assume it in the same terraform

Reply:
@davidkelliott Yeah I was looking at the output and realised that now, thanks